Editor: fix welcome pane scaling under dpiaware

ericoporto opened the pull request:
Allow scaling of the Welcome Pane, this is step 1/7, partial of #2087

I plan to work on each pane slowly when I find time, in separate PRs, also because these panes have lots of specific details, I think it's better to work them one by one.

<details>
<summary>Editor at 125% scaling</summary>

![image](https://github.com/adventuregamestudio/ags/assets/2244442/6b7e7ccd-821e-4fc5-8ead-c9b24af31963)

</details>


<details>
<summary>Editor at 200% scaling</summary>

![image](https://github.com/adventuregamestudio/ags/assets/2244442/402b3660-e8dc-458c-bf3f-5fd4385630d0)

</details>

To test this PR I recommend editing the `Editor/AGS.Editor/app.manifest` and uncommenting the dpi aware setting (lines 52 to 58).

ivan-mogilko commented:
Something I noticed: if I repeatedly click "Show another tip", sometimes changing a tip resizes the tip panel horizontally, and the big "what's new" panel shifts to right and left. This was not happening previously.

ericoporto commented:
Can you take a screenshot of this? Does it become larger?

ivan-mogilko commented:
I don't think a screenshot will tell much, the difference is tiny, but the shifting is noticeably if you repeatedly click on it.

It looks like the left panel becomes a bit wider, and the main panel becomes a bit smaller.

ericoporto commented:
> The question here is mostly why are they allowed to resize like this?

I made so they could grow horizontally a little too if necessary. But now I removed this and fixed the width, so it can only grow vertically. It appears to work alright at least from 100% to at most 300%. I think this if fine.
